src/App.js

```diff
@@ -1,21 +1,22 @@
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import Navbar from './Components/Navbar';
+import CountriesCountainer from './Pages/CountriesContainer';
+import Region from './Pages/RegionContainer';
+
+const App = () => (
+  <Router basename={process.env.PUBLIC_URL}>
+    <header>
+      <Navbar />
+    </header>
+    <Routes>
+      <Route path="/" element={<CountriesCountainer />} />
+      <Route path="/:country" element={<Region />} />
+    </Routes>
```

Comment:
[OPTIONAL] I would recommend removing `basename={process.env.PUBLIC_URL}` from the `Router` and adding `exact` to the first Route so the app will render the CountriesCountainer as a default page with the path `/`. When I open `localhost:3000/` it gives an empty page, I should add `/capstone-3` to be able to see the CountriesCountainer page.
```js
<Router>
  <Route exact path="/" element={<CountriesCountainer />} />
</Router>
```